doc/how-to/getting_started_go.rst

```diff
@@ -1,256 +1,279 @@
-Deleting data
-********************************************************************************
 
-To delete a tuple, use ``connection.Delete``:
 
-.. code-block:: go
+.. _getting_started_go_updating_data:
 
-    resp, err = conn.Delete("tester", "primary", []interface{}{4})
+Updating data
+*************
 
-To delete all tuples in a space (or to delete an entire space), use ``Call``.
-We'll focus on this function in more detail in the
-:ref:`next <getting_started-go-stored-procs>` section.
+``NewUpdateRequest()`` can be used to update a tuple identified by the primary key as follows:
 
-To delete all tuples in a space, call ``space:truncate``:
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Update
+    :end-at: Updated tuple
+    :dedent:
 
-.. code-block:: go
+``NewUpsertRequest()`` can be used to update an existing tuple or insert a new one.
+In the example below, a new tuple is inserted:
 
-    resp, err = conn.Call("box.space.tester:truncate", []interface{}{})
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Upsert
+    :end-before: // Replace
+    :dedent:
 
-To delete an entire space, call ``space:drop``.
-This requires connecting to Tarantool as the ``admin`` user:
 
-.. code-block:: go
+In this example, ``NewReplaceRequest()`` is used to delete the existing tuple and insert a new one:
 
-    resp, err = conn.Call("box.space.tester:drop", []interface{}{})
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Replace
+    :end-at: Replaced tuple
+    :dedent:
+
+
+
+
+.. _getting_started_go_deleting_data:
+
+Deleting data
+*************
 
-.. _getting_started-go-stored-procs:
+``NewDeleteRequest()`` in the example below is used to delete a tuple whose primary key value is ``5``:
+
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Delete
+    :end-at: Deleted tuple
+    :dedent:
+
+
+
+.. _getting_started_go_stored_procedures:
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 Executing stored procedures
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To execute a stored procedure, use ``NewCallRequest()``:
+
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Call
+    :end-at: Stored procedure result
+    :dedent:
+
+
+.. _getting_started_go_closing_connection:
+
+Closing the connection
+~~~~~~~~~~~~~~~~~~~~~~
+
+The ``CloseGraceful()`` method can be used to close the connection when it is no longer needed:
+
+..  literalinclude:: /code_snippets/snippets/connectors/go/hello.go
+    :language: go
+    :start-at: // Close connection
+    :end-at: Connection is closed
+    :dedent:
+
+..  NOTE::
```

Comment:
I'd put this note in the beginning of **Developing** subsection so that some readers can read the code instead of the procedure doc if they like.

Reply:
Usually, I place the links to full samples at the beginning and at the end of tutorials. Note that the first link already added right after the topic's title.